src/context.ts

```diff
@@ -67,10 +68,10 @@ export class Context {
     readonly message?: any,
   ) {}
 
-  getData(options: { requiredOnly: boolean } = { requiredOnly: false }) {
+  getData(options: { includeOptionals: IncludeOptionals } = { includeOptionals: true }) {
```

Comment:
This is a breaking change, the function signature should not be changed.

Reply:
I am not sure why it is breaking change. Could you explain in detail?

Reply:
Because the Context class is exported and exposed to the public, so changing the signature of the function (in this case the parameters) is a breaking change.